src/main/java/de/thomas_oster/liblasercut/drivers/EpilogHelix.java

```diff
@@ -74,76 +83,18 @@ public EpilogHelix clone()
     return result;
   }
 
-  //We need this methods for XMLEncoder to work properly
   @Override
-  public boolean isAutoFocus()
-  {
-    return super.isAutoFocus();
-  }
-  
-  @Override
-  public void setAutoFocus(boolean b)
-  {
-    super.setAutoFocus(b);
-  }
-  
-  @Override
-  public void setBedHeight(double bh)
-  {
-    super.setBedHeight(bh);
-  }
-  
-  @Override
-  public double getBedHeight()
-  {
-    return super.getBedHeight();
-  }
-  
-  @Override
-  public void setBedWidth(double bh)
-  {
-    super.setBedWidth(bh);
-  }
-  
-  @Override
-  public double getBedWidth()
-  {
-    return super.getBedWidth();
-  }
-  
-  @Override
-  public void setHostname(String host)
-  {
-    super.setHostname(host);
-  }
-  
-  @Override
-  public String getHostname()
-  {
-    return super.getHostname();
-  }
-  
-  @Override
-  public int getPort()
-  {
-    return super.getPort();
-  }
-  
-  @Override
-  public void setPort(int p)
-  {
-    super.setPort(p);
-  }
-  
-  @Override
-  public boolean isHideSoftwareFocus()
-  {
-    return super.isHideSoftwareFocus();
-  }
+  public int estimateJobDuration(LaserJob job)
+  {
+    // TODO: Calibrate these RASTER settings.
+    double PX2MM_500DPI = Util.px2mm(1, 500);
```

Comment:
If this change is accepted I'll spend the time to try to calibrate these settings which produce very wrong estimates IMO.